.circleci/config.yml

```diff
@@ -54,7 +56,9 @@ jobs:
           name: Install python 3
           command: brew install python3
       - run: ./test.sh
-      - run: ./test.py
+      - run: |
+         - source emsdk_env.sh
```

Comment:
is this change required because of this PR?

Reply:
I should be able to revert that, now that https://github.com/emscripten-core/emscripten/pull/10935 has landed.

Technically I think the tests are wrong here because they should run the context in whic `emsdk_env` has been run.   i.e. the tests were not following the instructions here.